lib/node/storage.js

```diff
@@ -1,15 +1,57 @@
 /* eslint no-unused-vars: 0 */
+import {readFile, writeFile} from "fs";
 
-export const canStoreURLs = false;
+export const canStoreURLs = true;
 
-export function setItem(key, value) {
+export class DefaultStorage {
+  constructor (dir) {
+    this.dir = dir
+  }
 
-}
+  setItem (key, value) {
+    return new Promise((resolve, reject) => {
+      this._getData().then((data) => {
```

Comment:
Unfortunately, this is not secured against race conditions as this is not an atomic operation. Think of a situation where between the _getData and _writeData calls another node process modifies the file. Those changes would be lost. Maybe there is a module on NPM which already provides a simple file-based key-value storage?